models/multi_organ_segmentation/docs/README.md

```diff
@@ -0,0 +1,129 @@
+- 7: Pancreas
+
+Here is an example of output.
+
+![alt用テキスト](output_example.png)
+
+## **Scores**
+
+This model achieves the following Dice score on the validation data (our own split from the whole dataset):
+
+Mean Dice = 88.6%
+
+## MONAI Bundle Commands
+In addition to the Pythonic APIs, a few command line interfaces (CLI) are provided to interact with the bundle. The CLI supports flexible use cases, such as overriding configs at runtime and predefining arguments in a file.
+
+For more details usage instructions, visit the [MONAI Bundle Configuration Page](https://docs.monai.io/en/latest/config_syntax.html).
+
+#### Execute model searching:
+
+```
+python -m scripts.search run --config_file configs/search.yaml
+```
+
+#### Execute multi-GPU model searching (recommended):
+
+```
+torchrun --nnodes=1 --nproc_per_node=8 -m scripts.search run --config_file configs/search.yaml
+```
+
+#### Execute training:
+
+```
+python -m monai.bundle run --config_file configs/train.yaml
+```
+
+Please note that if the default dataset path is not modified with the actual path in the bundle config files, you can also override it by using `--dataset_dir`:
+
+```
+python -m monai.bundle run --config_file configs/train.yaml
+```
+
+#### Override the `train` config to execute multi-GPU training:
+
+```
+torchrun --nnodes=1 --nproc_per_node=8 \
+     -m scripts.search run \
+     --config_file configs/search.yaml
+```
+
+#### Override the `train` config to execute evaluation with the trained model:
+
+```
+python -m monai.bundle run \
+    --config_file "['configs/train.yaml','configs/evaluate.yaml']"
+```
+
+#### Execute inference:
+```
+python -m monai.bundle run --config_file configs/inference.yaml
+```
+
+
+#### Export checkpoint for TorchScript:
+
+```
+python -m monai.bundle ckpt_export network_def --filepath models/model.ts --ckpt_file models/model.pt --meta_file configs/metadata.json --config_file configs/inference.yaml
+```
+
+#### Execute inference with the TensorRT model:
+
+```
+python -m monai.bundle run --config_file "['configs/inference.yaml', 'configs/inference_trt.yaml']"
+```
+
+
+## **References**
+
+[1] He, Y., Yang, D., Roth, H., Zhao, C. and Xu, D., 2021. Dints: Differentiable neural network topology search for 3d medical image segmentation. In Proceedings of the IEEE/CVF Conference on Computer Vision and Pattern Recognition (pp. 5841-5850).
+
+​
+[2] Roth, Holger R., et al. "A multi-scale pyramid of 3D fully convolutional networks for abdominal multi-organ segmentation." International conference on medical image computing and computer-assisted intervention. Springer, Cham, 2018.
+​
+
+[3] Shen, Chen, et al. "Effective hyperparameter optimization with proxy data for multi-organ segmentation." Medical Imaging 2022: Image Processing. Vol. 12032. SPIE, 2022.
+
+
+
+## **License**
+Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at
```

Comment:
Hi @cshen0625 @holgerroth , the license file shown in: https://github.com/Project-MONAI/model-zoo/pull/275/files#diff-009b21cc28e45820d20ee05f45a2c33421d542c2f932c6da8f66bd46990c02f4R1 is different from this section. Should one of them be modified?